airflow/contrib/operators/ecs_operator.py

```diff
@@ -139,6 +140,10 @@ def _check_success_task(self):
             raise AirflowException(response)
 
         for task in response['tasks']:
+            if re.match(r'Host .+?\(.+?\) terminated\.', task.get('stoppedReason', '')):
```

Comment:
Maybe add a comment here to elaborate where this regex matches on. Do you know what other `stoppedReasons` might be? Just to be sure that it doesn't conflict with other reasons.

Reply:
> Do you know what other `stoppedReasons` might be?

That's a great question. The [only documentation I can find](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/stopped-task-errors.html) identifies "**_some_** possible reasons and their explanations," which doesn't give a lot of confidence that the list is complete.

I would like to edit that check to match "terminated" or "stopped" though, having seen that documentation, now.

Reply:
> I would like to edit that check to match "terminated" or "stopped" though, having seen that documentation, now.

Amended with that change and added a comment as you suggested, as well, @Fokko